components/alert/alert.css

```diff
@@ -0,0 +1,78 @@
+:root{
+    --alert-error: #f5c0c6;
+    --alert-success: #d0f0e1;
+    --alert-warning: #f4dfc8;
+    --alert-info: #bde3f7;
+    --alert-error-outline: #f25b6d;
+    --alert-success-outline: #468367;
+    --alert-warning-outline: #f4a653;
+    --alert-info-outline: #4f91b5;
+}
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.alert-group{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    align-items: center;
+    border: 1px solid grey;
+    margin: 1rem;
+    max-width: 37.5rem;  
+    
+}
+
+.alert{
+    position: relative;
+    display: flex;
+    flex-wrap: nowrap;
+    margin: 0.5rem;
+    padding: 0.5rem;
+    max-width: 34.5rem;
+    border-radius: 0.3rem;
+    width: 90%;
+    
+}
+
+.alert-warning{
+ background-color: var(--alert-warning);
+}
+
+.alert-warning > span:first-child{
+    color: var(--alert-warning-outline);
+}
+
+.alert-info{
+    background-color: var(--alert-info);
+}
+
+.alert-info > span:first-child{
+    color: var(--alert-info-outline);
+}
+
+.alert-success{
+    background-color: var(--alert-success);
+}
+
+.alert-success > span:first-child{
+    color: var(--alert-success-outline);
+}
+
+.alert-error{
+    background-color: var(--alert-error);
+}
+
+.alert-error > span:first-child{
+    color: var(--alert-error-outline);
+}
+
+.material-icons-outlined{
+    margin-right: 1rem;
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
```

Comment:
why have you applied flex here? I don't think it's needed. When commented this code out, there was no change in display.

Reply:
So that when the screen width is around 200px icon gets aligned to the center. Try it out once.